api/app/controllers/spree/api/v2/platform/wishlists_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Spree
+  module Api
+    module V2
+      module Platform
+        class WishlistsController < ResourceController
+          private
+
+          def model_class
+            Spree::Wishlist
+          end
+
+          def scope_includes
+            [:wished_items]
+          end
+
+          def permitted_resource_params
+            # TODO:
+            # Are we allowing the Platform API to set the user for a wishlist?
```

Comment:
@MatthewKennedy yes apps should have this possibility

Reply:
Noted.